### PR TITLE
Upgrade docs for dbt-trino v1.2.2

### PR DESCRIPTION
**Pull request by hovaesco**
## Prerelease docs
If this change is related to functionality in a prerelease version of dbt (delete if not applicable):
- [ ] I've added versioning components, as described in ["Versioning Docs"](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/versioningdocs.md)
- [ ] I've added a note to the prerelease version's [Migration Guide](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/docs/guides/migration/versions)

## Checklist
If you added new pages (delete if not applicable):
- [ ] The page has been added to `website/sidebars.js`
- [ ] The new page has a unique filename

If you removed existing pages (delete if not applicable):
- [ ] The page has been removed from `website/sidebars.js`
- [ ] An entry has been added to `_redirects`

### CODE REVIEW

File: website/docs/reference/warehouse-profiles/trino-profile.md [hunk 1]
```diff
@@ -95,12 +96,4 @@ A dbt-trino profile can be configured to run against Trino or Starburst using th
...
 | threads                        | How many threads dbt should use                                                                              | Optional (default is `1`)                                                                                        | `8`                              |
 | prepared_statements_enabled    | Enable usage of Trino prepared statements (used in `dbt seed` commands)                                      | Optional (default is `true`)                                                                                     | `true` or `false`                |
-| retries                        | Configure how many times a database operation is retried when connection issues arise                        | Optional (default is `3`)  
-## Caveats
-
-### Unsupported Functionality
-
-Due to the nature of Trino, not all core dbt functionality is supported. The following features of dbt are not implemented on Trino:
-
-1. [Snapshots](snapshots)
```
Comment (dataders) [inline website/docs/reference/warehouse-profiles/trino-profile.md:105]: totally makes sense to drop this unsupported section, but do is there anywhere where we say the converse, as in "Feature that are supported", even if all features are supported?

Comment (hovaesco) [reply website/docs/reference/warehouse-profiles/trino-profile.md:105]: I haven't seen such section in other adapters profile page. We can always introduce something like that. Do you have some ideas or did I miss that it's actually in other adapters profile page?

Comment (dataders) [reply website/docs/reference/warehouse-profiles/trino-profile.md:105]: ok then, no problem. let's merge this now and hit this point when i've done more research. i bet users are excited to use snapshots!

**Review by dataders — APPROVED** [review]
🤌 🤌 :shipit:

**Pull request merged**